tests/test_detect_target.py

```diff
@@ -47,98 +85,76 @@ def image_zidane():
     """
     Load Zidane image.
     """
-    image = cv2.imread(IMAGE_ZIDANE_PATH)
+    image = cv2.imread(str(IMAGE_ZIDANE_PATH))
     result, zidane_image = image_and_time.ImageAndTime.create(image)
     assert result
     assert zidane_image is not None
     yield zidane_image
 
+@pytest.fixture()
+def expected_bus():
+    """
+    Load expected bus detections.
+    Format: [confidence, label, x1, y1, x2, y2]
```

Comment:
Move this to `create_detections()` . Follow the Autonomy style convention for function docstrings with parameters.